types/kiosk-window.d.ts

```diff
@@ -98,6 +98,12 @@ declare namespace KioskBrowser {
     mountPoint?: string;
```

Comment:
In another part of this PR, I add to the `UsbDrive` type and the `get-usb-drives` ipc more fields relevant to formatting, like filesystem of the found usb drive, etc. Are you intentionally not pulling it over? If you don't care about telling people whether a USB is already in the right format or not, I don't know if you'd need it.

Reply:
right, that part of the PR also changes the call to kiosk-browser, so I thought it would not be necessary for the simplest "format usb stick." Obviously your version is better, just trying to keep this change small.

Reply:
cool, just checkint